packages/template-retail-react-app/app/utils/url.js

```diff
@@ -115,7 +115,7 @@ export const productUrlBuilder = (product) => encodeURI(`/product/${product.id}`
  * @param {string} searchTerm
  * @returns {string}
  */
-export const searchUrlBuilder = (searchTerm) => `/search?q=${searchTerm}`
+export const searchUrlBuilder = (searchTerm) => "/search?q=" + encodeURIComponent(searchTerm)
```

Comment:
Minor but the linter would like the " replaced with '.

We may also want to update the associated url.test.js class so that it has a test that checks that the encoding is being applied to `&` in the search terms.

Reply:
Thanks for making this change!